test/test_jit.py

```diff
@@ -4329,6 +4329,34 @@ def forward(self, x):
         self.assertEqual(7, w(3))
         self.assertFalse("training" in w.state_dict())
 
+    def test_torchbind(self):
+        def test_equality(f, cmp_key):
+            obj1 = f()
+            obj2 = torch.jit.script(f)()
+            return (cmp_key(obj1), cmp_key(obj2))
+
+        def f():
+            val = torch.classes._TorchScriptTesting_Foo(5, 3)
+            val.increment(1)
+            return val
+        test_equality(f, lambda x: x)
+
+        with self.assertRaisesRegex(RuntimeError, "Expected a value of type 'int'"):
+            val = torch.classes._TorchScriptTesting_Foo(5, 3)
+            val.increment('foo')
```

Comment:
I don't see the bindings for Object, I am assuming they are already a part of a previous PR.